ext/lmdb-safe/lmdb-typed.hh

```diff
@@ -474,7 +573,7 @@ public:
     {
       typename Parent::cursor_t cursor = (*d_parent.d_txn)->getCursor(std::get<N>(d_parent.d_parent->d_tuple).d_idx);
 
-      std::string keystr = keyConv(key);
+      std::string keystr = makeCombinedKey(keyConv(key), MDBInVal(""));
```

Comment:
Ah, so the ID part of the combined key might be empty? How does that work?

Reply:
this is for a range search. Given a key "X", the following keys might exist:

* X|20
* X|634
* X|1231231

this keystr is the prefix that finds all those.